cylc/flow/prerequisite.py

```diff
@@ -334,7 +342,7 @@ def get_target_points(self):
             get_point(p) for p in self.iter_target_point_strings()
         ]
 
-    def get_resolved_dependencies(self) -> List[str]:
+    def get_satisfied_dependencies(self) -> List[str]:
```

Comment:
[Vaguely interesting historical aside: I think use of "resolved" here goes all the way back to Cylc 2, when we first got the ability to visualize a graph - not from the workflow definition because there was no workflow definition, just a bunch of task definition files with inputs and outputs - from dependencies "resolved" at runtime.]